### ISSUE TITLE
Job max attempts (cloud tasks vs $tries)

### ISSUE THREAD

**Issue by mbreithu-wtd** [stackkit/laravel-google-cloud-tasks-queue#79]
Since V3.3 now ignores the backoff config from the task queue and uses the backoffs defined on the Laravel job, should the max retry count on the job also work like it does for normal laravel jobs? Currently it seems that the max retries value from the cloud tasks queue is used to determine a rescheduling of the job....

**Comment by marickvantuil** [comment]
I agree, that would make more sense. Might start working on that soon!
